### PR TITLE
generate random poll daily

### PR DESCRIPTION
**Pull request by alonewerwolf**

### CODE REVIEW

File: wa_tilda_parser.py [hunk 1]
```diff
@@ -246,6 +251,46 @@ def filter(self, message):
...
+        is_anonymous=True,
+        allows_multiple_answers=False,
+    )
+
+
+def callback_daily(context):
+    context.bot.send_message(chat_id=context.job.context, text='Згенерувати опитування?',
+                             reply_markup=ReplyKeyboardMarkup(BUTTONS, resize_keyboard=True, one_time_keyboard=True))
+
+def set_daily_message(update, context):
```
Comment (BorysDrozhak) [inline wa_tilda_parser.py:274]: would be great to have some comments of what functions are dong. that can save a lot of time for readers in future

File: wa_tilda_parser.py [hunk 2]
```diff
@@ -246,6 +251,46 @@ def filter(self, message):
...
+    chat_id = update.message.chat_id
+    context.job_queue.run_daily(callback_daily, time=datetime.time(hour=14, minute=00, tzinfo=pytz.timezone('Europe/Kiev')),
+                                days=(0, 1, 2, 3, 4, 5, 6), context=chat_id, name=str(chat_id))
+
+
+def stop_daily(update, context):
```
Comment (BorysDrozhak) [inline wa_tilda_parser.py:280]: hy do we need stop?

File: utils/poll_data.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
+POLLS = [
+        {
+            'title': 'Вам подобається у WA?',
```
Comment (BorysDrozhak) [inline utils/poll_data.py:4]:
```suggestion
            'title': 'Яка опція безпесна для людини з лактозонепереносимістю',
```

File: utils/poll_data.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+            'questions': ['Так', 'Ні']
```
Comment (BorysDrozhak) [inline utils/poll_data.py:5]:
```suggestion
            'questions': ['Стей під гратеном', 'Стейк лосося під пті птімом', 'Грушевий тост', 'тунець татакі з авакадо']
```

File: utils/poll_data.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+        },
+        {
+            'title': 'Більше онігірьок?',
```
Comment (BorysDrozhak) [inline utils/poll_data.py:8]:
```suggestion
            'title': 'Яка з онігірьок підходить вегетеріанцям?',
```

File: utils/poll_data.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+            'questions': ['Так', 'Ні']
```
Comment (BorysDrozhak) [inline utils/poll_data.py:9]:
```suggestion
            'questions': ['Тунець', 'Лосось', 'Обидві',  'Жодну']
```

File: utils/poll_data.py [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+        },
+        {
+            'title': 'Філе міньйона круть?',
```
Comment (BorysDrozhak) [inline utils/poll_data.py:12]:
```suggestion
            'title': 'Який з алергенів часто може призвести до смерті?',
```

File: utils/poll_data.py [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+            'questions': ['Так', 'Ні']
```
Comment (BorysDrozhak) [inline utils/poll_data.py:13]:
```suggestion
            'questions': ['Лактоза', 'Перець', 'Мед', 'Глютен']
```

File: utils/poll_data.py [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+            'title': 'Яка з онігірьок підходить вегетеріанцям?',
+            'questions': ['Тунець', 'Лосось', 'Обидві',  'Жодну']
+        },
+        {
+            'title': 'Який з алергенів часто може призвести до смерті дорослої людини?',
+            'questions': ['Лактоза', 'Перець', 'Мед', 'Глютен']
+        },
+]
+
+BUTTONS = [['Так', 'Ні']]
```
Comment (BorysDrozhak) [inline utils/poll_data.py:17]: how these buttons work?

**Review by BorysDrozhak — APPROVED** [review]

File: wa_tilda_parser.py (hunk 2 above)
Comment (alonewerwolf) [reply wa_tilda_parser.py:294]: just in case if needed

**Pull request merged**